Makefile

```diff
@@ -12,23 +12,32 @@
 ## See the License for the specific language governing permissions and
 ## limitations under the License.
 
+SHELL := /bin/bash
+
+.PHONY: build
 build:
-	@bazel build ...:all
+	@bazel build //...
 
+.PHONY: clean
 clean:
-	@bazel clean
+	@bazel clean //...
 
+.PHONY: test
 test:
-	@bazel test ...
+	@bazel test //...
 
+.PHONY: lint
 lint: build
 	@bin/linters.sh
 
+.PHONY: fmt
 fmt:
 	@bin/fmt.sh
 
+.PHONY: coverage
 coverage:
 	@bin/codecov.sh
 
+.PHONY: racetest
```

Comment:
Do you mean something like ```.PHONY: build test lint``` ? If so this would have the side-effect of running test and lint whenever you call ```make build```